example/spec/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -85,14 +81,30 @@ describe('The S3 Ingest Granules workflow', () => {
   const granuleModel = new Granule();
   process.env.ExecutionsTable = `${config.stackName}-ExecutionsTable`;
   const executionModel = new Execution();
+  process.env.CollectionsTable = `${config.stackName}-CollectionsTable`;
+  const collectionModel = new Collection();
+  process.env.ProvidersTable = `${config.stackName}-ProvidersTable`;
+  const providerModel = new Provider();
   let executionName;
 
   beforeAll(async () => {
+    const collectionJson = JSON.parse(fs.readFileSync(`${collectionsDir}/s3_MOD09GQ_006.json`, 'utf8'));
+    const collectionData = Object.assign({}, collectionJson, {
+      name: collection.name,
+      dataType: collectionJson.dataType + testSuffix,
+    });
+
+    const providerJson = JSON.parse(fs.readFileSync(`${providersDir}/s3_provider.json`, 'utf8'));
+    const providerData = Object.assign({}, providerJson, {
+      id: provider.id,
+      host: config.bucket,
+    });
+
     // populate collections, providers and test data
     await Promise.all([
       uploadTestDataToBucket(config.bucket, s3data, testDataFolder),
-      addCollections(config.stackName, config.bucket, collectionsDir, testSuffix),
-      addProviders(config.stackName, config.bucket, providersDir, config.bucket, testSuffix)
+      apiTestUtils.addCollection({ prefix: config.stackName, collection: collectionData }),
```

Comment:
Is this actually testing the real collections endpoint? I think it's testing the testutils we have to set up tests.

Reply:
This function (and the rest of the utilities in `apiTestUtils`) mimic hitting the real endpoint by directly invoking the Lambda function backing that endpoint in the API Gateway, which in this case is the `ApiCollectionsDefault` lambda. 

What additional tests would we need to add? Are you referring to the `APISuccessSpec.js` ?

Reply:
I suppose? In my mind testing the endpoint involved directly hitting the endpoint. But we can bring this up in the meeting and see what people think? This might be good enough

Reply:
`apiTestUtils.addCollection()` is invoking the lambda as expected, so this is fine.